loader/checkpoint_test.go

```diff
@@ -113,9 +113,12 @@ func (t *testCheckPointSuite) TestForDB(c *C) {
 	c.Assert(count, Equals, len(cases))
 
 	// update checkpoints
-	conn, err := createConn(t.cfg)
+	db, conns, err := createConns(tctx, t.cfg, 1)
 	c.Assert(err, IsNil)
-	defer closeConn(conn)
+	conn := conns[0]
+	defer func() {
+		db.Close()
```

Comment:
better to check the error returned from `db.Close()` is nil

Reply:
sure, [Done](https://github.com/pingcap/dm/pull/266/files#diff-35fd8fc6b33e86de2e97c1ccef05bbdaR120-R121)